FlightReservation/components/signupView.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Text, 
+} from 'react-native';
+import ModalCustom from './modalComponent';
+import SignUpButton from './signUpButtons';
+import Form from './formComponent';
+import Terms from './termsComponents';
+
+function SignupForm(props) {
+  const {signUpGoogle,validEmail,setValidEmail, validPassword, setValidPassword, setIsLoginFormActive,isLoginFormActive,showObj, showModal, addFill, formObjectState, modalVisible, singedText, isIconCheck} = props;
```

Comment:
line too long, please make it shorter or user `Prettier`

Reply:
Done!